.github/workflows/tests.yml

```diff
@@ -9,16 +9,13 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
```

Comment:
I removed it, because it is not needed now, and because of version control it is not a problem to add it again as soon as needed and possible.

However, I added it again (just for version 3.11) so that the reintroduction is simplified later.